stash_api/app/controllers/stash_api/datasets_controller.rb

```diff
@@ -175,6 +175,7 @@ def lock_down_admin_only_params
       # all this bogus return false stuff is to prevent double render errors in some circumstances
       return if check_superuser_restricted_params == false
```

Comment:
I suppose we could combine these into one if statement with || between them, but not a big deal.